daemon/gamemode-context.c

```diff
@@ -602,10 +602,12 @@ int game_mode_context_query_status(GameModeContext *self, pid_t client, pid_t re
  */
 static GameModeClient *game_mode_client_new(pid_t pid, char *executable)
 {
+	/* clang-format off */
```

Comment:
Why is this needed (and the other one)? These are in my preferred style already but is it not possible to get clang-format to do this?

Reply:
I'm not 100% sure why but on multiple different installs, clang reformats these two sections differently to the Travis instance the CI is running. I've tried to coax it into doing it correctly with options but nothing seems to work.